extensions/amp-story/0.1/amp-story.css

```diff
@@ -926,3 +926,126 @@ amp-story[standalone].i-amphtml-story-landscape .i-amphtml-story-no-rotation-ove
 .i-amphtml-story-button-container {
   display: none !important;
 }
+
+/* Default loading spinner */
+
+.i-amphtml-story-spinner {
+  display: inline-block;
+  position: absolute;
+  top: calc(100% - 36px);
+  right: 12px;
+  width: 24px;
+  height: 24px;
+  z-index: 10;
+}
+
+.i-amphtml-story-spinner-container {
+  width: 100%;
+  height: 100%;
+  direction: ltr;
+}
+
+.i-amphtml-story-spinner[active] .i-amphtml-story-spinner-container {
+  animation: container-rotate 1294ms linear infinite;
```

Comment:
The fact that this is 1294ms instead of 1300ms makes me so curious...

Reply:
It's some weird trigonometry done from the base speed (1100ms), I can show you offline :)